### ISSUE TITLE
*Phpactor* buffer keeps popping up

### ISSUE THREAD

**Issue by kermorgant** [emacs-php/phpactor.el#27]
Hello,

As put in the title, the *Phpactor* buffer keeps popping up when I use this package, although I've disabled debug. I usually work with a 2 buffers side by side, and *Phpactor* pops up in the inactive one. A bit annoying as I have to get into that buffer and close *Phpactor* buffer quite often.

I'm not using the package from melpa but instead use load the files from the source code cloned locally,  so maybe my issue could stem from that ?

Any advice about how to disable that behaviour ?

**Comment by kermorgant** [comment]
Hi,

Judging from the annoying buffer name "\*Phpactor Output\*", this behaviour seems to come from [this line](https://github.com/emacs-php/phpactor.el/blob/f3b0e3d1e162b4f4fcdfe512bbab793c4808b358/phpactor.el#L123). 

Am I the only one experiencing this ?

**Comment by kermorgant** [comment]
@zonuexe what's your opinion about this change, would you be ok for a PR switching to ``call-process-region`` ?

**Comment by ejmr** [comment]
@zonuexe 

I agree with @kermorgant's proposal.  Often we will need temporary buffers to parse and handle the output of commands, but will not need to show them to user.  We should avoid showing such buffers, when they have no relevant information for the user.  See my code in the `php-auto-yasnippets` project for an example of using a temporary buffer containing output from an externel program, and the user never sees that bufffer.
